### ISSUE TITLE
Directory 'bin' missing

### ISSUE THREAD

**Issue by xnopre** [lesfurets/git-octopus#41]
I try to go to `test` directory, to run 1 test like this : `./run_test.sh simple_test` but the `../bin` is missing. Idem for `run_all.sh`. Thanks

**Comment by apflieger** [comment]
Which version are you using ? These tests will be dropped in 2.0 if I release the version some day...

**Comment by xnopre** [comment]
@apflieger I have cloned the master 3 days ago, I think this is the current version. I met Dimitri in Agile Grenoble, he's interested with my external tests, and to start, I'm very interested with the tests :-)

**Comment by apflieger** [comment]
You might need to run `make build` before running tests. It will create the bin directory

**Comment by xnopre** [comment]
Yes, you're right, thanks and sorry, it's explained in README but not very explicit for me to run the tests ;-) . But after `make build`, the test `git octopus -v` does not work : `git: 'octopus' is not a git command. See 'git --help'.`...

**Comment by apflieger** [comment]
Having the command available in git requires `make install`
